ads/directadvert.js

```diff
@@ -23,10 +24,28 @@ import {loadScript, validateData} from '../3p/3p';
 export function directadvert(global, data) {
   validateData(data, ['blockId']);
```

Comment:
Do you want to add server-name as an optional attribute here?

Reply:
Is this necessary?

Reply:
it is not required.